example/classifier/odp_classifier.c

```diff
@@ -993,6 +995,7 @@ static int parse_pmr_policy(appl_args_t *appl_args, char *optarg)
 		mask_sz = parse_custom(token, stats[policy_count].rule.mask_be, ODPH_ETHADDR_LEN);
 		if (mask_sz != ODPH_ETHADDR_LEN) {
 			ODPH_ERR("Invalid mask. Provide mask without 0x prefix.\n");
+			free(pmr_str);
```

Comment:
Instead of adding several `free()` calls it could be cleaner to add `goto err;` to each error branch and handle the return value and `free()` in one place. The `exit(EXIT_FAILURE);` lines could also be replaced with the same pattern.

Reply:
Error handling with `goto` added.